src/test/java/de/funfried/netbeans/plugins/external/formatter/sql/jsqlformatter/JSQLFormatterWrapperTest.java

```diff
@@ -29,14 +29,15 @@ public static void setUpClass() {
 	public void testFormat() {
 		final String text = "SELECT FOO FROM BAR WHERE FOO = 'BAR' ORDER BY FOO LIMIT 1";
 		final String expected = "SELECT foo\n" +
-				"FROM bar\n" +
-				"WHERE foo = 'BAR'\n" +
-				"ORDER BY foo\n" +
-				";";
+								"FROM bar\n" +
```

Comment:
The formatting seems to be wrong. Could you please try to use the Eclipse Formatter for Java with the template contained in the main folder of the sources?